src/dataflow/src/render/reduce.rs

```diff
@@ -275,7 +279,18 @@ where
         // If hierarchical, we can repeatedly digest the groups, to minimize the incremental
         // update costs on relatively small updates.
         if hierarchical {
-            partial = build_hierarchical(partial, &func)
+            if monotonic && is_min_or_max(&func) {
+                use differential_dataflow::operators::iterate::Variable;
+                use differential_dataflow::operators::Consolidate;
+                let delay = std::time::Duration::from_nanos(10_000_000_000);
```

Comment:
I just want to double check my understanding of the tradeoffs here -- if we shorten this delay then would timely and differential end up doing more work, but the memory footprint over a stream of inputs would be smaller? 

its to reconcile because the computation seems like it would be the same just in different sized chunks but maybe theres a overhead to having this delay be smaller that I don't know about

Reply:
I think the trade-off is that if we shorten the delay, then we can perform less concurrent work. With it set at 10s, we can retire 10s worth of updates at a time. If we lowered it to 10ms, then even if presented with 1s of updates we would have to retire them in 100 batches of 10ms.

Reply:
Got it thanks :)